src/ContentInfo/index.tsx

```diff
@@ -0,0 +1,100 @@
+import { useMemo } from "react";
+import { Navigate, useNavigate, useParams } from "react-router-dom";
+
+import { jsonSchemaValidator } from "@jaam-schema/src";
+
+import { useProjectQuery } from "../@shared/useProjectQuery";
+import { ContentPropertyList } from "./ContentPropertyList";
+import { TimeInfo } from "./TimeInfo";
+import {
+  useContentsState,
+  useSetContentsState,
+} from "../ProjectContents/useContentsStore";
+import { useUpdateContentMutation } from "../ProjectContents/useContentsMutation";
+import * as css from "./index.css";
+
+export function ContentInfo() {
+  const navigate = useNavigate();
+  const params = useParams();
+  const { projectName, schemaName, contentId } = params;
+  const { content: contentState } = useContentsState();
+  const { reset } = useSetContentsState();
+
+  if (!projectName || !contentId || !schemaName) {
+    return <Navigate to="/error" />;
+  }
+
+  const { data: project } = useProjectQuery(projectName);
+
+  if (!project) {
+    return <Navigate to="/error" />;
+  }
+
+  const { schemaList, storageKey: token } = project;
+  const schema = useMemo(
+    () => schemaList.filter(data => data.schemaName === schemaName)[0].schema,
```

Comment:
```suggestion
    () => schemaList.find(data => data.schemaName === schemaName).schema,
```
find가 더 적절한 메서드일 것 같습니다!